operator/images/access-setup/content/bin/setup_work_dir.sh

```diff
@@ -157,8 +157,8 @@ tekton_chains_manifest(){
       --workdir /workspace \
       --entrypoint /usr/bin/cosign \
       "$cosign_image" generate-key-pair
-    kubectl create namespace tekton-chains --dry-run=client -o yaml > "$chains_namespace"
-    kubectl create secret generic -n tekton-chains signing-secrets --from-file="$chains_tmp_dir" --dry-run=client -o yaml | \
+    kubectl create namespace openshift-pipelines --dry-run=client -o yaml > "$chains_namespace"
```

Comment:
why did you change the namespace here @Roming22 if in the tekton config you've switched the namespace to tekton-chains

Reply:
That's me missing the change when doing the rollback. Fixed.